lobby_server.yaml

```diff
@@ -14,7 +15,7 @@
 - version: "1.10"
   host: lobby.triplea-game.org
   port: 3304
-  http_server_uri: lobby.triplea-game.org:4567
+  http_server_uri: "http://lobby-staging.triplea-game.org:4567"
```

Comment:
Didn't we agree on the prerelease DNS?
Or am I referring to something different?

Reply:
Checked in namecheap, unless I missed it, the prerelease dns is not even there anymore : |

Reply:
Looks like you missed it
![Proof](https://user-images.githubusercontent.com/8350879/50709061-e1a66d80-1066-11e9-800f-2d18ce0c7d68.png)

After this is removed, can lobby-staging be removed, and an AAAA record be added for pre-release?

Reply:
Hmm, I must have missed it  :thinking: 

```
$ nslookup prerelease.triplea-game.org
Server:		127.0.0.53
Address:	127.0.0.53#53

Non-authoritative answer:
Name:	prerelease.triplea-game.org
Address: 45.79.16.133
Name:	prerelease.triplea-game.org
Address: 2600:3c00::f03c:91ff:fe1e:2b30
```

Reply:
Switched to: "prerelease.triplea-game.org" with: 5d28c5f87

Reply:
> After this is removed, can lobby-staging be removed, and an AAAA record be added for pre-release?

I'm not aware of any dependencies on 'lobby-staging', so I think answer is yes

Reply:
I removed the DNS entries for `lobby-staging` (just now) and added the AAAA record for `prerelease` (an hour ago)